catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/AbstractDurableFileConsumer.java

```diff
@@ -103,9 +106,13 @@ Exchange getExchange(File file, WatchEvent.Kind<Path> fileEvent, String referenc
     return exchange;
   }
 
-  private static class ErrorLoggingSynchronization implements Synchronization {
-    private final String file;
+  void submitExchange(Exchange exchange) {
```

Comment:
Can `submitExchange`, `class ErrorLoggingSynchronization`, `class ExchangeHelper`, and `getMetacardIdFromReference` be `private`?